themes/prologue/prologue.rst

```diff
@@ -2,26 +2,23 @@
 Prologue
 ********
 
-The default PrestaShop install offers a neutral theme in black and white, enabling sellers to quickly and freely start their activity, whatever their business line.
+The default PrestaShop installation comes with one simple theme styled in black and white. This neutral style allows any seller to begin selling right away.
 
-Nearly 2000 themes are available through the PrestaShop Addons marketplace (http://addons.prestashop.com/en/). They were created either by members of the PrestaShop community or the PrestaTeam, and are sold at reasonable prices – some are even free.
+There are over 2,000 themes available through the [**PrestaShop Addons Marketplace**] (http://addons.prestashop.com/en/). Created by members of the PrestaShop community or the PrestaTeam. And sold at reasonable prices – some are even free.
 
-As a graphic designer/web developer, you too can put your themes up for sale on the PrestaShop Addons site, and earn 70% of the selling price.
+As a graphic designer or web developer, you too can put your themes up for sale on the PrestaShop Addon's site.  And earn 70% of the retail price.
 
 
 
 Migrating from PrestaShop 1.6
 =============================
 
-PrestaShop 1.7 introduces a totally reworked theme system from version 1.6, with the goal of making PrestaShop upgrades easier, and of making it easier/faster to create a brand new theme.
+PrestaShop 1.7 introduces a new theme system. Which is quite different from its predecessor in 1.6. There are no simple methods to migrate your 1.6 theme to 1.7
 
-The huge change to the theme system means that there is no easy way to migrate your PrestaShop 1.6 theme to PrestaShop 1.7.
+During this automatic upgrade, your theme will be replaced with **Classic Theme**. Which is the new default theme. So we recommend you begin work on your 1.7 theme changes before you make the switch to PrestaShop 1.7!
```

Comment:
Not just "1.7 theme changes": the user has to wholly port her 1.6 theme to 1.7, rewrite it all, not just update it.